account_statement_import_online/wizards/online_bank_statement_pull_wizard.xml

```diff
@@ -28,6 +28,12 @@
                         default_focus="1"
                         class="btn-primary"
                     />
+                    <button
+                        name="action_debug"
+                        string="Debug"
```

Comment:
I think the label on this one can be a little bit more descriptive (accountants also use debug mode to see more features, not just developers). Maybe `Show transaction data` plus a more explanatory help text on this one and then replace `Pull` on the other button by something like `Import transactions`?

Reply:
My suggestions here: https://github.com/OCA/bank-statement-import/pull/664/commits/6d104bd597bbde65a3542c81e365acb5b4b4e665